test/module/irohad/torii/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ target_link_libraries(torii_queries_test
     server_runner
     processors
     query_execution
+    common_execution
```

Comment:
Move to `irohad/execution/CMakeLists.txt` for `target_link_libraries(query_execution` because common execution is used in query executor impl.